src/app/components/scan-fingers/scan-fingers.component.html

```diff
@@ -1 +1,37 @@
-<p>scan-fingers works!</p>
+<div
+  class="scanner__wrapper"
+  [ngStyle]="{
+    'border-color': !isReaderConnected
+      ? 'gray'
+      : scanStatus === 'Successful'
```

Comment:
create an enum for all statuses we have

Reply:
@hoangngk Use ScanStatus for status of scanning and StatusColor for return the color depending on scan status.